test/e2e/cypress.json

```diff
@@ -2,6 +2,7 @@
   "baseUrl": "http://127.0.0.1:4000",
   "viewportWidth": 1366,
   "viewportHeight": 768,
+  "defaultCommandTimeout": 10000,
```

Comment:
Let's keep an eye on this. If we can avoid high timeouts, I believe it'd be nice :smile: